app/views/partials/_recovery_code.html.slim

```diff
@@ -8,7 +8,8 @@
   .navy.monospace
     - code.split(' ').each do |word|
       p.bold.mb0.h3
-        = word
+        span.recovery-code-word
```

Comment:
Since this class isn't used for styling, maybe we could prefix it with something like `spec-` so it's clear what it's being used for? We currently do something similar for `js-` when using classes as javascript selectors.  This class could probably also just be added to the `p` element instead of adding a new `span`.